src/macros.jl

```diff
@@ -85,103 +87,44 @@ Usage:
 
 Return `result` for the first matching `pattern`.
 If there are no matches, throw `MatchFailure`.
-This uses a brute-force code gen strategy, like using a series of if-else statements.
+This uses a brute-force code gen strategy, essentially a series of if-else statements.
 It is used for testing purposes, as a reference for correct semantics.
+Because it is so simple, we have confidence about its correctness.
 """
 macro __match__(value, cases)
     handle_match_cases_simple(__source__, __module__, value, cases)
 end
 
-"""
-Usage:
-```
-    @match value begin
-        pattern1 => result1
-        pattern2 => result2
-        ...
-    end
-```
-
-Return `result` for the first matching `pattern`.
-If there are no matches, throw `MatchFailure`.
-"""
+#
+# """
+# Usage:
+# ```
+#     @match value begin
+#         pattern1 => result1
+#         pattern2 => result2
+#         ...
+#     end
+# ```
+
+# Return `result` for the first matching `pattern`.
+# If there are no matches, throw `MatchFailure`.
+# """
+#
```

Comment:
I would bring back the proper docstrings here

Reply:
See above.  I could put them in this file, but not on every declaration.